src/math/bessel.hh

```diff
@@ -199,15 +199,6 @@ namespace rascal {
         if (n_up > 0) {
           this->upward_recursion(distance, fac_a, n_up);
         }
-
-        // set small values to 0.
```

Comment:
this was not doing anything, since unaryExpr does not modify data inplace.

Reply:
Removing this introduce no changes in behaviour. If you want to make sure the data is modified, this should be a separate PR, to be discussed separately =)

Reply:
It is a one line change so please undo the delete at the very least.

Reply:
I can undo the delete if someone takes care of it in the next days/weeks. Else it will stay like this for god know how much longer, inducing confusion in readers.

Do you want to take care of this change?

Reply:
yes please